tests/test_pin_job.py

```diff
@@ -18,7 +18,7 @@ def test_pin_job(self, base_url, selenium):
         assert job.symbol == page.pinboard.selected_job.symbol
 
     def test_pin_job_from_job_details(self, base_url, selenium):
-        # Open treeherder page, select first job, pin it by the pin icon
+        """ Open treeherder page, select next job, pin it by the logviewer icon"""
```

Comment:
I'm not sure I understand this change. The test selects the first job (next seems ambiguous), and does not use a logview icon at all.